Mage/src/main/java/mage/game/GameImpl.java

```diff
@@ -1843,15 +1862,24 @@ public void addEmblem(Emblem emblem, MageObject sourceObject, Ability source) {
      */
     @Override
     public void addEmblem(Emblem emblem, MageObject sourceObject, UUID toPlayerId) {
+        logger.info("Copying...");
```

Comment:
Is this logger info useful to release or just for debugging?

Reply:
Forgot to take those out, good catch